Pod/Public/TCCAnimationTileOverlayRenderer.m

```diff
@@ -50,6 +54,10 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
     }
 }
 
+- (int)tccTileSizeForZoomLevel:(int)zoomLevel {
+    return (int)(TCCTileSize * pow(2, (TCCMaxZoomLevel - zoomLevel)));
```

Comment:
i would have expect the tile size to reduce on zooming out. here it seems to be increasing. ex. it doubles when zoom level goes from 20 to 19. can you please help me understand this?

Reply:
Renaming this method to make it a bit more clear.

Reply:
discussed with @jterhorst that the size increases because it corresponds to lat/lon size